dataactcore/config_example.yml

```diff
@@ -52,10 +52,16 @@ broker:
     # the broker_files directory specified above
     rss_folder: rss
     rss_file: rss_file.xls
-
+    
     # S3 filenames for SF-133 file, only required if planning to load SF-133 table
     sf_133_folder: config
 
+    # The path where the broker will store sf133 files and emails
+    # when the broker is not using an AWS S3 bucket.
+    # This should be an absolute path. Note that this path
+    # is ignored if use_aws is true.
+    sf_133_files: /full/path/to/sf133/files
```

Comment:
Where do you see this? The existing import code, if there is no path specified and `use_aws` is false, returns an empty array.